eng/tools/oav-runner/src/cli.ts

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/env node
+
+import { checkSpecs, checkExamples } from "./runner.js";
+import {
+  outputAnnotatedErrors,
+  outputErrorSummary,
+  outputSuccessSummary,
+  ReportableOavError,
+} from "./formatting.js";
+import { consoleLogger } from "@azure-tools/specs-shared/logger";
+
+import { parseArgs, ParseArgsConfig } from "node:util";
+
+export async function main() {
+  const config: ParseArgsConfig = {
+    options: {},
+    allowPositionals: true,
+  };
+
+  const { positionals } = parseArgs(config);
+
+  const [targetDirectory, runType] = positionals;
+
+  if (!targetDirectory) {
+    console.error("Error: <targetDirectory> is required.");
+    process.exit(1);
+  }
+
+  if (runType !== "specs" && runType !== "examples") {
+    console.error("Error: <runType> must be either 'specs' or 'examples'.");
+    process.exit(1);
+  }
+
+  consoleLogger.info(
+    `Running oav-runner against ${runType} within ${targetDirectory}.`,
+  );
+
+  let exitCode = 0;
+  let scannedSwaggerFiles: string[] = [];
+  let errorList: ReportableOavError[] = [];
+  let reportName = "";
+
+  if (runType === "specs") {
+    [exitCode, scannedSwaggerFiles, errorList] =
+      await checkSpecs(targetDirectory);
+    reportName = "Swagger Specifications Validation";
+  }
+
+  if (runType === "examples") {
```

Comment:
```suggestion
  } else if (runType === "examples") {
```
Nit: I think this is clearer that exactly one is executed